Library/Homebrew/livecheck/strategy/sparkle.rb

```diff
@@ -146,6 +144,33 @@ def self.items_from_content(content)
           end.compact
         end
 
+        # Filters out items that aren't suitable for Homebrew.
+        #
+        # @param items [Array] appcast items
+        # @return [Array]
+        sig { params(items: T::Array[Item]).returns(T::Array[Item]) }
+        def self.filter_items(items)
+          items.select do |item|
+            # Omit non-macOS items
+            next false if item.os && !((item.os == "osx") || (item.os == "macos"))
```

Comment:
`!((item.os == "osx") || (item.os == "macos"))` is a bit hard to parse, would be nice to split it out into variables or similar

Reply:
If we add an array constant for this (`MACOS_STRINGS = ["macos", "osx"].freeze`), we could use `if item.os && MACOS_STRINGS.none?(item.os)` instead. Is that more understandable or did you have something else in mind?

Reply:
They're common values in [Sparkle appcasts](https://sparkle-project.org/documentation/publishing/#alternate-download-locations-for-other-operating-systems). Since this is already in a `Sparkle` class (and `SPARKLE_` would feel repetitive), how about `APPCAST_MACOS_STRINGS`?

Reply:
That's fine, just a comment above the variable explaining that would be handy!